packages/salesforcedx-vscode-apex-debugger/src/adapter/debugConfigurationProvider.ts

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Moved this out of `index.ts` and added lineBreakpoint & workspace info as part of the configuration.